homeassistant/components/zwave_js/discovery.py

```diff
@@ -1125,36 +1125,29 @@ def async_discover_single_configuration_value(
             ConfigurationValueType.RANGE,
             ConfigurationValueType.MANUAL_ENTRY,
         ):
-            if value.metadata.type == ValueType.BOOLEAN or (
-                value.metadata.min == 0 and value.metadata.max == 1
```

Comment:
Why do we no longer check the metadata min and max?

Reply:
the new `ConfigurationValueType.BOOLEAN` enum gets returned based on the equivalent of this check for min and max: https://github.com/home-assistant-libs/zwave-js-server-python/blob/master/zwave_js_server/model/value.py#L305-L306

but it doesn't handle the case where the value type is `boolean` in the metadata because that value will not have a min and max